myapp/account-service.js

```diff
@@ -0,0 +1,22 @@
+function AccountService(){
+  this.username=data.username;
+  this.email=data.username;
+  this.password=data.password;
+}
+
+// function register(){
+//   var username = document.forms["registerForm"]["username"].value;
+//   var mail = document.forms["registerForm"]["email"].value;
+//   var pas = document.forms["registerForm"]["password"].value;
+//
+// }
+document.addEventListener("DOMContentLoaded", function(event){
+  var x= localStorage.getItem("userData");
+  if( x !== null){
+    document.getElementById('searchLogOut').visibility = 'visible';
```

Comment:
get searchLogOut and searchLogIn elements only once at the beginning of the function. Put them in well named variables